demo/README.md

```diff
@@ -1,13 +1,35 @@
 # ev3dev demo programs
 
-This folder contains a bunch of demo programs that you can use to help you in
+This folder contains several demo programs that you can use to help you in
 developing your own code. Brief descriptions of each demo are provided below;
 you can access the full source code and some more detailed information on each
 by opening the respective folders above.
 
+To install these on your EV3, use git to clone the ev3dev-lang-python repository.
```

Comment:
Given that we're presenting two independent options here, can you start this with something like "There are two options for running these demo programs. We recommend that you..."?